### DISCUSSION TITLE
Commit 0ec6e10 works :)

### DISCUSSION THREAD

**Discussion by Benman2785** [JohnImril/diablo_web_simple#21] [Q&A] [answered]
i just wanted to give feedback -> my branch added some stuff for me -> but i will still test all your new versions (as i did with Commit 0ec6e10) = it works fine :)

I hope you have a wonderful christmas and a good start into 2025

**Comment by JohnImril** [comment]
Hi, I am very pleased that you are following the updates. I will continue to try to keep the repository up to date. I wish you a happy Christmas and a happy new year!!! And I would be grateful if you would close the discussions that you made in the projects with a note that the answer was found (if you have time, of course) maybe this way the GitHub algorithms will be able to show the project to more users

P.S. I subscribed to you so as not to miss your activity

**Comment by Benman2785** [comment]
i will do that now :)

i will also subscribe to you.

Enjoy Christmas and have a happy New Year :)

**Comment by Benman2785** [comment]
the answer was found

**Comment by JohnImril** [comment]
Isn't this what a closed and answered discussion should look like? Here's an example:   
![image](https://github.com/user-attachments/assets/cecfe5fc-51c6-4af0-9c28-5081940bf90e)

**Reply by JohnImril** [reply]
I just never closed it and I don't know. Thank you very much if you explain.

**Reply by Benman2785** [reply]
actually i am not sure -> i am not that deep into github (yet)

Merry Christmas by the way :)

**Comment by JohnImril** [comment, marked as answer]
![image](https://github.com/user-attachments/assets/99347215-5f0c-4af2-a7b7-265af6188a00)
Hi, do you have the ability to mark as answer now? I think it was all about the category

**Reply by Benman2785** [reply]
now i can mark it as answer :)

**Reply by JohnImril** [reply]
Thanks a lot
